CHANGES.md

```diff
@@ -89,11 +89,6 @@
 
 # [2.58.0] - Unreleased
 
-## Highlights
-
-* New highly anticipated feature X added to Python SDK ([#X](https://github.com/apache/beam/issues/X)).
-* New highly anticipated feature Y added to Java SDK ([#Y](https://github.com/apache/beam/issues/Y)).
-
 ## I/Os
 
 * Support for [Solace](https://solace.com/) source (`SolaceIO.Read`) added (Java) ([#31440](https://github.com/apache/beam/issues/31440)).
```

Comment:
This sounds like a long-anticipated addition, so worth making a highlight.

Reply:
Thanks, let's reflect the changes here also in CHANGES.md on master after this is merged.